src/main/java/com/contentful/java/cma/ModuleLocales.java

```diff
@@ -0,0 +1,323 @@
+   *
+   * @param retrofit the instance to use the service with.
+   * @return a populated instance of the service to be used.
+   */
+  @Override protected ServiceLocales createService(Retrofit retrofit) {
+    return retrofit.create(ServiceLocales.class);
+  }
+
+  /**
+   * Use all functionalities of this module asynchronously.
+   *
+   * @return an asynchronous handling instance.
+   */
+  public Async async() {
+    return async;
+  }
+
+  /**
+   * Fetch all locales of this space.
+   *
+   * @param spaceId the space identifier identifying the space.
+   * @return the array of locales.
+   * @throws IllegalArgumentException if spaceId is null.
+   */
+  public CMAArray<CMALocale> fetchAll(String spaceId) {
+    assertNotNull(spaceId, "spaceId");
+    return service.fetchAll(spaceId).blockingFirst();
+  }
+
+  /**
+   * Fetch specific locales of this space.
+   *
+   * @param spaceId the space identifier identifying the space.
+   * @param query   the search criteria to search for.
+   * @return the array of locales.
+   * @throws IllegalArgumentException if spaceId is null.
+   */
+  public CMAArray<CMALocale> fetchAll(String spaceId, Map<String, String> query) {
+    assertNotNull(spaceId, "spaceId");
+    if (query == null) {
+      return service.fetchAll(spaceId).blockingFirst();
+    } else {
+      return service.fetchAll(spaceId, query).blockingFirst();
+    }
+  }
+
+  /**
+   * Fetches one locale by its id from Contentful.
+   *
+   * @param spaceId the space this locale is hosted by.
+   * @param localeId  the id of the locale to be found.
+   * @return null if no locale was found, otherwise the found locale.
+   * @throws IllegalArgumentException if space id is null.
+   * @throws IllegalArgumentException if locale id is null.
+   */
+  public CMALocale fetchOne(String spaceId, String localeId) {
+    assertNotNull(spaceId, "spaceId");
+    assertNotNull(localeId, "localeId");
+
+    return service.fetchOne(spaceId, localeId).blockingFirst();
+  }
+
+  /**
+   * Create a new locale.
+   *
+   * @param spaceId the space id to host the locale.
+   * @param locale    the new locale to be created.
+   * @return the newly created locale.
+   * @throws IllegalArgumentException if space id is null.
+   * @throws IllegalArgumentException if locale is null.
+   */
+  public CMALocale create(String spaceId, CMALocale locale) {
+    assertNotNull(spaceId, "spaceId");
+    assertNotNull(locale, "locale");
+
+    final CMASystem sys = locale.getSystem();
+    locale.setSystem(null);
+
+    try {
+      return service.create(spaceId, locale).blockingFirst();
+    } finally {
+      locale.setSystem(sys);
+    }
+  }
+
+  /**
+   * Update the given locale instance on Contentful.
+   * <p>
+   * Please make sure that the instance provided is fetched from Contentful. Otherwise you will
+   * get an exception thrown.
+   *
+   * @param locale the locale fetched from contentful, updated by caller, to be updated.
```

Comment:
There are some instances, like here, where _Contentful_ is not capitalized, as it should be 🙂